audit.py

```diff
@@ -334,15 +468,15 @@ def __init__(self, started, output, include_archived_repositories, include_forke
         self.include_archived_repositories = include_archived_repositories
         self.include_forked_repositories = include_forked_repositories
 
-    def _table_header(self):
-        headers = [
+    def _table_header(self, headers: list):
```

Comment:
```suggestion
    def _table_header(self, headers: list[tuple[str, dict[str, str]]]):
```

Maybe define a type for header so you can write:

```suggestion
    def _table_header(self, headers: list[Header]):
```

Reply:
Followed your lead, applied throughout